src/DotNet.Glob/DotNet.Glob.csproj

```diff
@@ -27,6 +27,10 @@
     <AssemblyOriginatorKeyFile>..\..\Dotnet.Glob.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
- 
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net4' And '$(TargetFramework)' != 'netcoreapp2.1' ">
+    <PackageReference Include="System.Memory">
```

Comment:
This package does not provide an AsSpan() method.
It will be used optional by a `net452` project (let's call it Net452Prj) that uses this package.
If you want to use AsSapn() in Net452Prj, do the following: 
1. Run the `Install-Package System.Memory` from Package Manager Console.
2. Call `AsSapn()` on the string or array before calling `IsMatch`.

Reply:
Ok thanks.

One small thing, would you mind changing the msbuild ItemGroup condition to list the TFs that the package is required for, rather than the ones it isn't necessary for. For example:

```
Condition=" '$(TargetFramework)' == 'net45' OR '$(TargetFramework)' == 'net46' OR '$(TargetFramework)' == 'netstandard1.1' "
```

The reason is, I expect over time, this way will lead to lessmaintainence, for example when `net50` comes along, we won't have to modify this condition